releasenotes/notes/tensored_op_gradient_fix-34a08b8b4c012ad6.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
As opflow has been moved from Aqua, there will be a migration guide created and a complete single release doc created. So for this release these individual release note yaml files are not needed. I.e. this can be removed.

Reply:
Thanks for the comment.